README.md

```diff
@@ -1,16 +1,235 @@
-# Outline
-
-1. Database structure (columns, rows, etc)
-2. How to create table
-3. How to alter table
-4. How to delete table
-
-all of the above should be a code along
-
 # Database Basics 
 
 ## Objectives
 
 1. Understand database structure. 
 2. Learn about the different types of data you can store in a database and how. 
-3. Learn how to create, alter and delete database tables. 
+3. Learn how to create, alter and delete database tables.
+
+## Database Structure
+
+Relational Databases like SQLite store data in a structure we refer to as a table. You can think of a table in a database a lot like you would a spreadsheet. We define specific columns in our table, and then we store any number of what we refer to as 'records' as rows in our database. A record is just information referring to one specific entity. For instance, if you had a table called "People" you could imagine a structure like this:
+
+| name | age | email |
+|------|-----|-------|
+| Bob | 29 |  bob@flatironschool.com |
+| Avi  |  28   | avi@flatironschool.com  |
+| Adam |  28   | adam@flatironschool.com |
+
+Each column has a name, and each row contains the corresponding information about a person.
+
+Looking at this table, see if you can answer some simple questions:
+
+How many rows do we have?
+What is the age of the person in the second row?
+Find the record with the name Avi, what is the email for that record?
+
+These are some of the kinds of questions SQL allows us to ask our database about this table, which we will cover later. 
+
+### Note on Column Names
+
+When we name columns in our database, there are a couple conventions we will follow. The first is that we will always use lowercase letters when refering to columns in our database. SQLite isn't case sensitive about its commands or column names, as we will discuss below, but it is general best practice for us to stick to lowercase for our column names.
+
+The Second convention we want to follow is more important. That is that when we have multiple words in a column name we link the together use underscores rather than spaces. We call this convention "snake_case". So, for instance, if we wanted to be more specific with our email column above, we would have called it something like email_address. If we wanted to split up name to first and last we might have columns called first_name and last_name.
+
+## Database Tables
+
```

Comment:
no set up required. this is a code along and they are instructed to work in the command line. the prev. reading had them install sqlite3 if they didn't have it already.